goosebit/updater/controller/v1/routes.py

```diff
@@ -34,11 +34,11 @@ async def polling(request: Request, dev_id: str, updater: DeviceUpdateManager =
     if device is None:
         raise HTTPException(404)
 
-    sleep = config.poll_time_default
+    sleep = config.poll_time
 
     if device.last_state == UpdateStateEnum.UNKNOWN:
-        # device registration
-        sleep = config.poll_time_registration
+        # device registration: force device to poll again in 10s. After registration, an update might be available
+        sleep = "00:00:10"
```

Comment:
May want to keep this configurable just in case?  I suppose easy enough to change later...

Reply:
To me it is more of an internal decision of gooseBit to know what is a reasonable time to wait between registration and checking for an update. So don't want to hand over the burden to the user through a config. If we see a need in the future to have different settings we can bring it back.

Reply:
We may want to just move this to a constant in settings then, but this works.